src/Console/Command/FixCommand.php

```diff
@@ -347,6 +347,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
+        if ($config->usingCache()) {
+            $cacheFile = $config->getCacheFile();
+            if (file_exists($cacheFile) && null !== $stdErr) {
```

Comment:
this even echo'ed into `assertFile` of PHPUnit, liked to have that one fail for directories even though those are files of some form  ;)